From ac63fc4a38eed52b71a022c2a8f367a7ca7ff891 Mon Sep 17 00:00:00 2001
From: beardypig <beardypig@users.noreply.github.com>
Date: Tue, 15 Nov 2016 15:42:12 +0000
Subject: [PATCH] support for relative path to rtmpdump

---
 src/streamlink/stream/rtmpdump.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/streamlink/stream/rtmpdump.py b/src/streamlink/stream/rtmpdump.py
index 7ffe3e0..fed3c1c 100644
--- a/src/streamlink/stream/rtmpdump.py
+++ b/src/streamlink/stream/rtmpdump.py
@@ -1,4 +1,5 @@
 import re
+import os.path
 
 from time import sleep
 
@@ -22,7 +23,7 @@ class RTMPStream(StreamProcess):
     def __init__(self, session, params, redirect=False):
         StreamProcess.__init__(self, session, params)
 
-        self.cmd = self.session.options.get("rtmp-rtmpdump")
+        self.cmd = os.path.realpath(self.session.options.get("rtmp-rtmpdump"))
         self.timeout = self.session.options.get("rtmp-timeout")
         self.redirect = redirect
         self.logger = session.logger.new_module("stream.rtmp")
-- 
2.10.2

